profiles/functions

```diff
@@ -474,14 +474,16 @@ restore_logs_syncing() {
 }
 
 irqbalance_banned_cpus_clear() {
-    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance
+    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance && \
+        test ${1:-restart} = restart && systemctl try-restart irqbalance
```

Comment:
Could you please rewrite this to 'if' (the part starting with the 'test')? We didn't define API for functions, but I would take it the way that the function can return non zero exit code in case of some problem. This would return non zero (error) exit code even when there is no error and the argument is just e.g. 'norestart' which could have other unintended consequences - e.g. imagine situation when this function would be used as the last one in the script.sh (without the explicit return 0), which could cause the script.sh to return non zero exit code in case everything is OK.

Reply:
Makes sense.